src/http/one/Tokenizer.cc

```diff
@@ -78,32 +62,17 @@ Http::One::Tokenizer::qdText(SBuf &returnedToken, const bool http1p0)
              */
             static const CharacterSet qPairChars = CharacterSet::HTAB + CharacterSet::SP + CharacterSet::VCHAR + CharacterSet::OBSTEXT;
             SBuf escaped;
-            if (!prefix(escaped, qPairChars, 1)) {
-                returnedToken.clear();
-                restoreLastCheckpoint();
-                return false;
-            }
-            returnedToken.append(escaped);
-            continue;
+            if (!tok.prefix(escaped, qPairChars, 1))
```

Comment:
This seems inconsistent or even wrong: If the next symbol is invalid, then we should not (silently) return false as if we are simply waiting for more data. We probably should throw an error. 

N.B. It does not matter whether the old code was broken here as well -- we are changing too much of this code to use that as an excuse IMO.

Reply:
The throwing condition is wrong AFAICT: false prefix() does not necessarily mean we found an invalid character. It may also mean that we found no characters at all.

Reply:
Done (781dbe5).

Reply:
FTR because this is extremely confusing to see done with zero change. The 'done' status appears to be because the atEnd() check is done immediately after the quote is detected. So the tricky case never reaches this call to prefix().